### PR TITLE
Remove info about restricted tokens

### PR DESCRIPTION
**Pull request by TomWFox**
parse-community/parse-server#6612

### CODE REVIEW

**Comment by mtrezza** [comment]
Should the info about [restricted session token](https://docs.parseplatform.org/js/guide/#session--security) also be removed from the docs?

**Comment by TomWFox** [comment]
I think so, not certain about these changes, but there is my attempt.

**Comment by mtrezza** [comment]
Why not certain?

**Comment by TomWFox** [comment]
I guessed a bit about what parts should/shouldn't be removed. I basically took out anything that seemed to be about restricted tokens but some of what I removed may still be relevant without restricted tokens idk.

**Comment by mtrezza** [comment]
Correction: I added the wrong bookmark tag in my previous post. I actually meant to ask you to remove the session property "[restricted](https://docs.parseplatform.org/js/guide/#session-properties)".

**Comment by mtrezza** [comment]
Correction: I added the wrong bookmark tag in my previous post. I actually meant to ask you to remove the session property "[restricted](https://docs.parseplatform.org/js/guide/#session-properties)".

**Comment by mtrezza** [comment]
Correction: I added the wrong bookmark tag in my previous post. I actually meant to ask you to remove the session property "[restricted](https://docs.parseplatform.org/js/guide/#session-properties)".

**Comment by mtrezza** [comment]
Correction: I added the wrong bookmark tag in my previous post. I actually meant to ask you to remove the session property "[restricted](https://docs.parseplatform.org/js/guide/#session-properties)".

File: _includes/common/sessions.md [hunk 1]
```diff
@@ -294,26 +291,8 @@ try {
...
 
 * **Find**, **Delete** — Useful for building a UI screen that allows users to see their active session on all devices, and log out of sessions on other devices. If your app does not have this feature, you should disable these permissions.
-* **Create** — Useful for "Parse for IoT" apps (e.g. Arduino or Embedded C) that provision restricted user sessions for other devices from the phone app. You should disable this permission when building apps for mobile and web. For "Parse for IoT" apps, you should check whether your IoT device actually needs to access user-specific data. If not, then your IoT device does not need a user session, and you should disable this permission.
+* **Create** — Useful for apps that provision user sessions for other devices from the phone app. You should disable this permission when building apps for mobile and web. For "Parse for IoT" apps, you should check whether your IoT device actually needs to access user-specific data. If not, then your IoT device does not need a user session, and you should disable this permission.
```
Comment (mtrezza) [inline _includes/common/sessions.md]: Can we change the `"Parse for IoT" apps` term? I think this is rather confusing, I wouldn't know what this terms is referring to specifically. I think this simply means Parse that is used in an IoT environment?

**Comment by mtrezza** [comment]
> I guessed a bit about what parts should/shouldn't be removed. I basically took out anything that seemed to be about restricted tokens but some of what I removed may still be relevant without restricted tokens idk.

I’ve looked through it, it looks good to me

File: _includes/common/sessions.md (hunk 1 above)
Comment (TomWFox) [reply _includes/common/sessions.md]: I agree...

`For "Parse for IoT" apps` -> `For IoT apps`

**Review by mtrezza — APPROVED** [review]

**Pull request merged**